tests/test_cv.py

```diff
@@ -278,7 +278,6 @@ def test_retrieve_cv_folds():
                                                  stratified=False,
                                                  cv_folds=num_folds,
                                                  grid_search=False,
-                                                 grid_objective='f1_score_micro',
```

Comment:
1. We allow it because a lot of the time people want to run the same experiment just toggling grid search to false (when it was true before) and taking out the objectives would be an extra step that they would need to remember to do. We do [check this](https://github.com/EducationalTestingService/skll/blob/master/skll/config.py#L794) and raise a n appropriate warning and basically ignore the objectives going forward. Does that make sense?
2. Good catch on `cross_validate`. I will fix it to be consistent with train.

Reply:
@mulhod please see my answer for 1 and review the latest changes for 2. And re-review. Thanks!